fastn-core/src/http.rs

```diff
@@ -687,3 +686,81 @@ pub async fn github_graphql<T: serde::de::DeserializeOwned>(
 
     Ok(return_obj)
 }
+
+/// construct `ftd.http` consumable error responses
+/// https://github.com/fastn-stack/fastn/blob/7f0b79a/fastn-js/js/ftd.js#L218C45-L229
+/// ```json
+/// {
+///     data: null,
+///     errors: {
+///         key: Vec<String>
+///     }
+/// }
+/// ```
+pub async fn user_err(
+    errors: Vec<(&str, Vec<&str>)>,
```

Comment:
The code in `ftd.js` handles both `errors: Vec<(&str, Vec<&str>)>,` and `errors: Vec<(&str, &str)>,`, and I think we never send a list of errors for same keys, its a provision django has, but I have never used. So lets simplify our lives by seeing single string for each key.

Reply:
[e90639](https://github.com/fastn-stack/fastn/pull/1612/commits/e90639e9666b3b5cc867d590efb56e82afc6e846)